### PR TITLE
Update BOSH to 268+ / Xenial 97

### LINKED ISSUE #42: Upgrade: BOSH 268+
As a person who (a) uses BOSH, (b) uses Genesis, (c) needs more [Hacktoberfest PRs](https://hacktoberfest.digitalocean.com/stats/jhunt), and (c) creates issues in Github...

I would like to use runtime config placement rules that involve instance_group specifications.  This is reportedly only available in BOSH Directors from 268 onward.

### PR DESCRIPTION
**Pull request by jhunt**
Fixes #42

### CODE REVIEW

File: manifests/bosh/bosh.yml [hunk 1]
```diff
@@ -89,6 +95,18 @@ instance_groups:
 
     agent:
       mbus: (( concat "nats://" instance_groups.bosh.properties.nats.user ":" instance_groups.bosh.properties.nats.password "@" params.static_ip ":4222" ))
+      env:
+        bosh:
+          blobstores:
+            - provider: dav
+              options:
+                # FIXME: one day we should set this to https://, for full TLS
+                endpoint: (( concat "http://" params.static_ip ":252520" ))
```
Comment (Proplex) [inline manifests/bosh/bosh.yml]: This should be `25250`

Comment (jhunt) [reply manifests/bosh/bosh.yml]: Fixed in ca300dc

**Comment by jhunt** [comment]
Just pushed the proto changes for using Xenial stemcells.  We'll try this out tomorrow in buffalo-lab proto-BOSH

**Comment by jhunt** [comment]
my test proto-BOSH director was redeployed successfully this morning without incident.

**Pull request merged**